panacea/web/static/js/app.js

```diff
@@ -46,12 +46,15 @@ async function handleDrugsResponse(response) {
       View.displayNoDrugsError();
     }
 
+    // fetching the pml here, TODO: decide when is best to do this
+    const pml = await Request.pml(ast);
```

Comment:
Same thing with whitespace - it all gets stripped. Submitting `no_drugs.pml` will highlight the wrong line as `unnamed` because there were blank lines in the source file that were stripped.